src/index.html

```diff
@@ -6,11 +6,48 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+      class="form-big"
+    >
+      <div class="icon-container">
+        <input
+          type="search"
+          name="searchBarBig"
+          class="search-bar sb-big"
```

Comment:
Use readable class names to not be confused when you have more classes in the future
```suggestion
          class="search-bar search-bar-big"
```